### PR TITLE
WIP demo flow documented

### PR DESCRIPTION
**Pull request by corinnekrych**

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -10,7 +10,66 @@ eval $(minishift docker-env)
 make test-e2e-local
 ```
 
-## Steps to verify operator registry
+## Steps to verify operator registry on OCP4
```
Comment (baijum) [inline test/README.md]: There is not much difference with the existing document other than a couple of namespaces.
https://github.com/redhat-developer/devconsole-operator/tree/master/test#steps-to-verify-operator-registry

File: make/dev.mk [hunk 2]
```diff
@@ -6,9 +6,10 @@ include ./make/git.mk
 
 DOCKER_REPO?=quay.io/openshiftio
 IMAGE_NAME?=devconsole-operator
-REGISTRY_URI=quay.io
+REGISTRY_URI?=quay.io
+REGISTRY_ORG?=redhat-developers
 
-DEVCONSOLE_OPERATOR_IMAGE?=quay.io/redhat-developers/devconsole-operator
+DEVCONSOLE_OPERATOR_IMAGE?=$(REGISTRY_URI)/$(REGISTRY_ORG)/$(IMAGE_NAME)
```
Comment (kwk) [inline make/dev.mk]: @corinnekrych are you using those variables somewhere else or do you absolutely need the ability to set the operator image name from the outside? I mean you could always add a tag to this image as well if you need to customize it. One lesson learned from the fabric8-wit project was to not introduce variables for things you rarely need.

File: test/README.md [hunk 3]
```diff
@@ -10,7 +10,66 @@ eval $(minishift docker-env)
...
+### 1. Build and push image to quay
+```
+QUAY_USERNAME=YYY QUAY_PASSWORD=XXX REGISTRY_ORG=YYY make push-operator-image
+```
+> NOTE: to test you can push to your own username in quay, therefore QUAY_USERNAME and REGISTRY_ORG are the same
+
+### 2. create a catalog 
+This catalog contains a link to the operator image you've just created in step1.
+* Create a new file `catalog.yaml`
+```
```
Comment (kwk) [inline test/README.md]:
```suggestion
```yaml
```

File: test/README.md [hunk 4]
```diff
@@ -10,7 +10,66 @@ eval $(minishift docker-env)
...
+* login to OCP4 
+```
+oc apply -f catalog.yaml
+oc get catsrc —all-namespaces
+```
+You should see your new catalog.
+
+### 3. create a subscription
+* Create a new file `subscription.yaml`
+```
```
Comment (kwk) [inline test/README.md]:
```suggestion
```yaml
```

**Comment by corinnekrych** [comment]
very good point @baijum @kwk 
let me close this PR

**Pull request closed without merging**